run.sh

```diff
@@ -30,8 +30,10 @@ while [[ $# > 0 ]]; do
 done
 
 # Create shared folders
-mkdir -p $SOURCE
-mkdir -p $CCACHE
+# Although docker would create non-existing directories on the fly,
+# we need to have them owned by the user (and not root), to be able
+# to write in them, which is a neccessity for startup.sh
+mkdir -p $SOURCE $CCACHE
```

Comment:
Would still like to have two `mkdir` commands used here. Don't merge commands only because it's possible... :-)